tests/test_exposure.py

```diff
@@ -1135,6 +1136,34 @@ def testExposureSummaryExtraComponents(self):
         self.assertEqual(summaryStats.ra, testDict['ra'])
         self.assertEqual(summaryStats.decl, testDict['decl'])
 
+    def testExposureSummarySchema(self):
+        """Test that we can make a schema for an exposure summary and populate
+        records with that schema.
+        """
+        schema = afwTable.Schema()
+        afwImage.ExposureSummaryStats.update_schema(schema)
+        self.maxDiff = None
```

Comment:
This makes `assertEquals` and the like actually show you what differs even when it's long.  It's something I added here while debugging a test failure originally, and I hadn't intended to leave it, but I now I wonder if I should, since I always find myself adding it when a test does fail and I need to investigate.